### ISSUE TITLE
Is pain trademarked?

### ISSUE THREAD

**Issue by Pampattitude** [peerigon/updtr#19]
Hey guys,

Great module!
However, I have a concern about the `README.md`. It may sound like a joke, but oh well.

The `README.md` states "Update outdated npm modules with zero pain™", but it's unclear what is (jokingly) trademarked, making it sound like "pain" is the focus of the so-called trademark. It might be interesting to rephrase it to avoid confusion.

Again, I'm not kidding! It's confusing and might degrade the image of the module.

Regards,

**Comment by alexlawrence** [comment]
:+1:

**Comment by alessioalex** [comment]
I believe pain is also patented somewhere by MegaCorp and the module author may be liable :dancers:

**Comment by jhnns** [comment]
Thx, I appreciate your honest feedback.

However, I don't think that this is an issue for most people, but I'll leave this open for discussion. If more people find this offending, I'll change it.

**Comment by slippycheeze** [comment]
FWIW, this would be the exact application of the symbol for a catchphrase, which is how I read the original.  So... you are perfectly in line with the legal use the symbol, and despite the confusion, not using it in an uncommon way.

Disclaimer: I'm not your lawyer, or a lawyer, but I have dealt with this question in the past with actual lawyers, which informs my comment here.
